src/explicitode.jl

```diff
@@ -302,10 +302,8 @@ function taylorinteg{T<:Number}(f, x0::T, t0::T, tmax::T, order::Int,
 
     # Integration
     while t0 < tmax
-        # xold = x0
-        # δt, x0 = taylorstep(f, t0, tmax, x0, order, abstol)
         δt, x0 = taylorstep!(f, xT, t0, tmax, x0, order, abstol)
-        xT = Taylor1( x0, order )
+        xT.coeffs[1] = x0
```

Comment:
Indeed, this may save some time, which accumulated may be significant! Yet, I'm not sure if it is not "dangerous", in the sense that for the next call of `taylorstep!` contains a `xT` with the correct constant value, but some *non-zero* coefficients for the rest. While this may be ok, since they are overwritten, it's work putting a comment there!